lib/orogen/templates/typekit/CMakeLists.txt

```diff
@@ -105,8 +105,8 @@ set_target_properties(${libname} PROPERTIES LINK_INTERFACE_LIBRARIES ${OrocosRTT
 set_target_properties(${libname} PROPERTIES INTERFACE_LINK_LIBRARIES ${OrocosRTT_LIBRARIES})
 if(WITH_RPATH AND APPLE)
   set_target_properties( ${libname} PROPERTIES
-    INSTALL_NAME_DIR "@rpath"
-    LINK_FLAGS "-Wl,-rpath ${CMAKE_INSTALL_PREFIX}/lib -rpath ${CMAKE_INSTALL_PREFIX}/lib/orocos -rpath ${CMAKE_INSTALL_PREFIX}/lib/orocos${OROCOS_PKG_DIR}/types")
```

Comment:
@smits these lines was added by you, are they still needed or was this a (old) nonworking workaround for cmake?